src/main/resources/application.properties

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.*/
 #
-
+spring.datasource.hikari.maximum-pool-size=50
```

Comment:
can we do it in the way so we can configure such parameters using environment variables?

Reply:
removed from properties